PerformanceTests/Sources/PerformanceTestScript/Util/Benchmark.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct BenchmarkResult: Codable {
+    let name: String
+    let description: String
+    let unit: String
+    let date: Int
+    let measurements: [Double]
+    let dimensions: [Dimension]?
+}
+
+struct Dimension: Codable {
+    let name: String
+    let value: String
+}
+
+struct BenchmarkReport: Codable {
+    let productId: String
```

Comment:
Should we just fix the value of `productId` to a string constant like `AWS SDK for Swift`?